pkg/k8s/api.go

```diff
@@ -221,3 +221,13 @@ func GetPodStatus(pod corev1.Pod) string {
 
 	return reason
 }
+
+// GetAddOnsConfigMap returns the data in a configmap
+func GetAddOnsConfigMap(kubeAPI kubernetes.Interface, namespace string) (map[string]string, error) {
```

Comment:
I don't see anything in this function that is specific to addons. Would this be better named `GetConfigMap`?

Reply:
Yeah, but we ain't passing the name of the add-on cm, right? So, It's not a general function? :thinking:

Reply:
So it's not the fact that we don't pass the name of the add-on configmap, it's that this function isn't specific to configmaps for *only* add-ons. If there were more configmaps in the namespace, would those be returned?

Reply:
I see that `ValuesConfigMapName` is Linkerd add-on specific. Nevermind my confusion about the naming on this!